server.go

```diff
@@ -1,24 +1,28 @@
 package main
 
 import (
+	"fmt"
 	"io"
 	"net"
 	"net/http"
 	"os"
+	"time"
 
+	"github.com/gorilla/context"
+
+	"github.com/getlantern/measured"
+
+	"./filters"
 	"./forward"
 	"./httpconnect"
-	"./profilter"
-	"./tokenfilter"
-	"./uidfilter"
 	"./utils"
 )
 
 type Server struct {
 	connectComponent     *httpconnect.HTTPConnectHandler
-	lanternProComponent  *profilter.LanternProFilter
-	tokenFilterComponent *tokenfilter.TokenFilter
-	uidFilterComponent   *uidfilter.UIDFilter
+	lanternProComponent  *filters.LanternProFilter
+	tokenFilterComponent *filters.TokenFilter
+	uidFilterComponent   *filters.UIDFilter
```

Comment:
Generally other languages fix these repetition issues with _class mixins_, inheritance or even generics. I wonder if we could respect an homogenous structure using structure embedding in Go. 
Anyway, code repetition is one of the warts of Go so we are pretty much bound to it unless we contort the structure to avoid repetition, like you are proposing here.